tests/performance/sum_up_failures.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
```

Comment:
Missing license 

Also please add some doc to the header about what this script does nad how can one use it.